spec/support/context/with_docker_container.rb

```
@@ -1,11 +1,22 @@
 shared_context 'with a docker container' do
+  def is_running?(container)
+    output = %x(docker inspect -f '{{.State.Health.Status}}' #{container}).chomp
+    if output == 'healthy'
```

This is a little too naive and as a result won't fail fast when the return value is `unhealthy`.

I think the wait loop needs to be moved into this block of code.